test/it/uk/gov/hmrc/exports/scheduler/jobs/PurgeAncientSubmissionsJobSpec.scala

```diff
@@ -0,0 +1,351 @@
+package uk.gov.hmrc.exports.scheduler.jobs
+
+import com.kenshoo.play.metrics.PlayModule
+import com.mongodb.client.MongoCollection
+import org.bson.Document
+import play.api.Application
+import play.api.inject.bind
+import play.api.inject.guice.GuiceApplicationBuilder
+import play.api.libs.json.{Format, Json}
+import uk.gov.hmrc.exports.base.{IntegrationTestPurgeSubmissionsToolSpec, TestExportsClient}
+import uk.gov.hmrc.exports.models.declaration.ExportsDeclaration.Mongo._
+import uk.gov.hmrc.exports.models.declaration.notifications.UnparsedNotification
+import uk.gov.hmrc.exports.mongo.ExportsClient
+import uk.gov.hmrc.exports.util.ExportsDeclarationBuilder
+import uk.gov.hmrc.mongo.workitem.{WorkItem, WorkItemFields}
+
+import scala.collection.JavaConverters._
+
+class PurgeAncientSubmissionsJobSpec extends IntegrationTestPurgeSubmissionsToolSpec with ExportsDeclarationBuilder {
```

Comment:
WOuld it be possible to add a test to ensure that correct order of deletion is enforced (unparsed first, parsed next etc)?